machine_configuration/README.md

```diff
@@ -87,6 +87,23 @@ $ lsmod | grep ast
 ```
 Ensure that ast driver is not loaded.
 
+# Machine Configuration for Provisioning Intel QAT
+
+* Create QAT machine config pool:
+
+Following command creates a custom machine config pool for worker nodes with Intel QAT feature. These nodes are labeled `intel.feature.node.kubernetes.io/qat: 'true'` by NFD.
+
+```$ oc apply -f https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/intel-qat-machine-config-pool.yaml```
+
+* Turn on intel_iommu kernel parameter for QAT
+
+For QAT, intel_iommu is a kernel parameter that needs to be turned on using the following command.
+
+```$ oc apply -f https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/100-intel-qat-intel-iommu-on.yaml```
```

Comment:
this could also add loading `vfio-pci` with the desired `ids` as parameters since we have to do the booting anyway.

Reply:
is it possible for us to add the ids as the parameter for the container and set it through the operator? So we don't have to hard code the ids in in the init-container? :-)